tests/subsys/usb/bos/src/test_bos.c

```diff
@@ -203,9 +203,9 @@ static void test_usb_bos_macros(void)
 
 	/* usb_bos_fix_total_length(); corrected with register */
 
-	hexdump((void *)hdr, len);
```

Comment:
I think line 206 should be changed as it is intended to dump real bos descriptor section.

Reply:
Agree here, we can move this line after len check.

Reply:
Moved all 3 hexdumps after length check